.env.sample

```diff
@@ -5,6 +5,8 @@ APP_ENV=development
 API_URL=http://drb-api-qa.nypl.org
 
 # PDF PROXY ENDPOINT
+# In development, refer to the instructions here to use the proxy: 
+# https://github.com/NYPL-Simplified/web-reader#cors-proxy
```

Comment:
I think we should include this on some sort of documentation for installing/setting up dev env

Reply:
Oh yeah!  just added a line in the readme about that